manifests/s/Slik/Subversion/1.14.2001/Slik.Subversion.locale.en-US.yaml

```
@@ -0,0 +1,13 @@
+# Created using wingetcreate 1.0.4.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.1.0.schema.json
+
+PackageIdentifier: Slik.Subversion
+PackageVersion: 1.14.2001
+PackageLocale: en-US
+Publisher: SlikSvn & The SharpSvn Project
+PackageName: Subversion 1.14.2

```

```suggestion
PackageName: Slik Subversion
```
to match Control Panel.